webpack.config.js

```diff
@@ -159,6 +159,7 @@ function appConfig(options, argv) {
                             test: /app\/polyfill\.ts$/,
                             include: [
                                     join(__dirname, 'packages', 'core', 'src'),
+                                    join(__dirname, 'packages', 'legacy-hoc', 'src'),
```

Comment:
Do we need to add this here?

Reply:
@animesh1987 Yes, because this package needed additional loader for `Injecthoc`

<img width="776" alt="Screenshot 2023-08-15 at 6 02 18 pm" src="https://github.com/bigcommerce/checkout-js/assets/141614330/6e07e9d7-a72c-493f-afd6-7a87b18dfd24">